### ISSUE TITLE
precommit-yaml : accept multi-document yaml file

### ISSUE THREAD

**Issue by geekobi** [pre-commit/pre-commit#635]
Hi,
I use a tool which uses a multi-document yaml file as configuration file. It doesn't accept several "mono-document" yaml files instead.
So, to use pre-commit, I disabled the yaml check and I would like to re-enable it.
Regards

**Comment by asottile** [comment]
Hi @geekobi, first thanks for the issue!

I think the specific hook you're referring to is `check-yaml` which is located [here](https://github.com/pre-commit/pre-commit-hooks) -- I'd suggest opening an issue there!

**Comment by asottile** [comment]
This is now available in the [v1.1.0 release of pre-commit-hooks](https://github.com/pre-commit/pre-commit-hooks/releases/tag/v1.1.0)

To use this, upgrade to `sha: v1.1.0` and use the following configuration for `check-yaml`:

```yaml
    -   id: check-yaml
        args: [--allow-multiple-documents]
```

Thanks again for the issue!

**Comment by geekobi** [comment]
thanks a lot
